configure.ac

```diff
@@ -19,20 +19,52 @@ AC_INIT(RcppArmadillo, m4_esyscmd_s([awk -e '/^Version:/ {print $2}' DESCRIPTION
 if test -z "${R_HOME}"; then
     AC_MSG_ERROR([Could not determine R_HOME.])
 fi
+
 ## Use R to set CXX and CXXFLAGS
+R=${R_HOME}/bin/R
```

Comment:
One can do this, and then use `R` as below, I found it helpful to to keep the longer `${R_HOME}/bin/R` to remind myself 
of the many time KH had to remind me to not use use `R` from the path.  So weak personal preference for the longer form.